sympy/simplify/tests/test_simplify.py

```diff
@@ -85,6 +86,58 @@ def test_ratsimpmodprime():
         y/2
 
 
+def test_rootofsimp():
+    r = RootOf(x**5 - 5*x + 12, 0)
```

Comment:
Maybe add a test or two to make sure it handles the different roots of the same polynomial correctly (like `r0 = RootOf(x**5 - 5*x + 12, 0)`, `r1 = RootOf(x**5 - 5*x + 12, 1)`, `rootofsimp(r0**5 - 5*r1 + 12)`).